core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -59,7 +59,17 @@ oppia.factory('explorationData', [
             $log.info('Retrieved exploration data.');
             $log.info(response.data);
 
-            explorationData.data = response.data;
+            // FIXME: Update this value based on what exactly is being sent from
```

Comment:
@526avijitgupta this does not look like something that should be in the final product...